src/styles/blocks/watch.scss

```diff
@@ -0,0 +1,58 @@
+.watch {
+  position: relative;
+  background-color: $bcg-clr;
+
+  @include square($size);
+  border: 5px solid;
+  border-radius: 25%;
+
+  &__seconds-arrow {
+    height: $sec-arrow-height;
+    width: $arrow-width;
+    background-color: $clock-border-clr;
+    border-radius: $arrow-bd-rad;
+
+    transform-origin: bottom;
+    animation: seconds 60s linear backwards infinite;
+
+    @extend %arrow-position;
+  }
+
+  &__minutes-arrow {
+    background-color: $clock-border-clr;
+    height: $min-arrow-height;
+    width: $arrow-width;
+    border-radius: $arrow-bd-rad;
+
+    transform-origin: bottom;
+    animation: minutes 3600s linear backwards infinite;
+
+    @extend %arrow-position;
+  }
```

Comment:
you have a part of repeatable styles here. 
I would recommend you to use just 
```
&__seconds-arrow, 
&__minutes-arrow {
  {{common_styles}}
}
```
and then use modifiers `--big` and `--small` to pass styles with a difference